wgsl/index.bs

```diff
@@ -4188,11 +4191,12 @@ in [[#texture-storage]].
 
 When the texel format does not have all four channels, then:
 
-* When reading the texel:
+* When reading the texel, the [=channel transfer function=] is applied [=component-wise=]:
```

Comment:
In the next table `Texel Formats for Storage Textures`, the spec includes the construction of the shader values. I did not include the inverse in that table as it wasn't entirely happy with how it was looking.

I would appreciate feedback on whether others think it should be included.

Reply:
I don't think we need to add it for CR.